Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -84,6 +89,19 @@ YUI.add('ez-actionbarview-tests', function (Y) {
             );
         },
 
+        _isActionDisabled: function (actionId) {
```

Comment:
@yannickroger  now it's used, because i applied your suggestion of adding the test to the existing suite. 
added the `_isRootLocation` method to the location model... is that correct? or were you meaning adding in the actionbarview itself?